projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid.navigation.spec.ts

```diff
@@ -157,6 +159,7 @@ describe('IgxHierarchicalGrid Navigation', () => {
         });
 
         it('should allow navigating to start in child grid when child grid target row moves outside the parent view port.', async () => {
+            hierarchicalGrid.navigateTo(2);
```

Comment:
Isn't this the same as the next line that does `scrollTo(2)`? There shouldn't be a need to call both. Just one or the other.